ios/FluentUI/Avatar/AvatarTokenSet.swift

```diff
@@ -30,11 +30,14 @@ public class AvatarTokenSet: ControlTokenSet<AvatarTokenSet.Tokens> {
         /// The gap around the ring around the `Avatar`.
         case ringOuterGap
 
-        /// The thickness of the outline around the presence icon.
-        case presenceIconOutlineThickness
+        /// The thickness of the outline around the presence/activity.
+        case borderThickness
```

Comment:
Is this going to be a breaking change? Is it necesary?

Reply:
The presence outline thickness and the activity outline thickness is intentionally the same. I felt like it was redundant to add another token, so I decided to create a generic borderThickness to address the two